source/scss/components/multioffer-teaser/structure.md

```diff
@@ -0,0 +1,49 @@
+```
```

Comment:
Just curious... What is this file doing?

Reply:
The same as the beginning of the multioffer-teaser.scss. It is moved from there.
It gives you an overview of the structure of this element.